compiler/rustc_symbol_mangling/src/export.rs

```diff
@@ -62,6 +62,8 @@ impl<'tcx> AbiHashStable<'tcx> for Ty<'tcx> {
             ty::Uint(uint_ty) => uint_ty.name_str().abi_hash(tcx, hasher),
             ty::Float(float_ty) => float_ty.name_str().abi_hash(tcx, hasher),
 
+            ty::Field(..) => todo!("FIXME(field_projections): no idea what to do here"),
```

Comment:
Not asking to change. I wonder if we can just move it down into the unsupported types for now.

Reply:
I haven't triggered an ICE through this todo, so maybe we can? I'll just let the other reviewers look over it.